frontend/src/cards/ui/MultiMapDialog.tsx

```diff
@@ -104,6 +104,9 @@ export function MultiMapDialog(props: MultiMapDialogProps) {
                     hideActions={false}
                     scaleType="quantile"
                     geoData={props.geoData}
+                    filename={`${props.metricConfig.fullCardTitleName}${
+                      breakdownValue === "All" ? "" : ` for ${breakdownValue}`
```

Comment:
This is generating "COVID-19 Cases Per 100K People in United Statesfalsefalse - Health Equity Tracker.png" for the total US map

Reply:
Issue was in MapCard.jsx; can't use short circuit evaluation inside a string template literal like: 
```
`${someBoolean && "print this if true"}` 
```
and instead need to use a ternary:
```
`${someBoolean ? "print this if true" : ""}`
```
to prevent JS from coercing a Boolean shortcircuit into a string (eg `false` into "false")